README.md

```diff
@@ -14,7 +14,8 @@ A program in Ruby that decrypts Morse Code into plain English.
 - **$ git clone https://github.com/...**
 - **$ cd ...**
 - **$ git checkout ...**
-- **$ run npm install to couple all dependacies**
+- **$ run bundle install to couple all dependacies**
+- **$ run ruby main.rb**
```

Comment:
- [ ] [OPTIONAL] If you want to make your readme file look even better it might be a good idea to put the commands and code between tilde (backtick) signs.

Reply:
thanks for the insight

Will do. 🙇